src/p5.dance.js

```diff
@@ -352,6 +352,10 @@ module.exports = class DanceParty {
     }
 
     var sprite = this.p5_.createSprite(location.x, location.y);
+    
+    if (name) {
+      sprite.name = name;
```

Comment:
Not sure if it's important, but at this point, a sprite isn't valid without a name since we could never reference it? Is there a way to error handle that case?

Reply:
There is a use case for anonymous dance sprites, since you can still interact with them by costume group. In that case, name would be `null` (see https://github.com/code-dot-org/dance-party/blob/91c92500b91c239ce2d6a90fcc1a555152156bb0/src/api.js#L41)